tests/node-e2e/outgoing/test_outgoing_traffic_many_requests.mjs

```diff
@@ -39,20 +40,27 @@ function makeRequests() {
       );
 
       response.on("data", (data) => {
-        process.stdout.write(`>> received ${data.slice(0, 4)}`);
+        console.log(`>> received ${data.slice(0, 4)}`);
       });
 
       response.on("error", (fail) => {
-        process.stderr.write(`>> response from ${host} failed with ${fail}`);
+        console.log(`>> response from ${host} failed with ${fail}`);
         throw fail;
       });
     });
 
     request.on("error", (fail) => {
-      process.stderr.write(
-        `>> request to ${requestIndex} ${host} failed with ${fail}`
-      );
-      throw fail;
+      console.log(`>> request to ${requestIndex} ${host} failed with ${fail}`);
+      if (fail.errno !== -104 || fail.code !== 'ECONNRESET') {
+        console.log("This is not a tolerated error, fail test.")
+        throw fail;
+      }
+      if (++failures > 1) {
+        console.log("Too many failed requests, failing test.");
+        throw fail;
+      } else {
+        console.log("Allowing one failed request (See https://github.com/metalbear-co/mirrord/issues/473)");
```

Comment:
The link isn't really necessary here - just briefly explain the reason

Reply:
Done